components/src/core/components/Input/_variables.scss

```diff
@@ -23,3 +24,17 @@ $oxd-checkbox-border-radius: 0.35rem;
 $oxd-checkbox-label-margin: 0.5em;
 $oxd-checkbox-checked-background-color: $oxd-primary-one-color;
 $oxd-checkbox-disabled-background-color: $oxd-interface-gray-lighten-2-color;
+
+$oxd-switch-width: 2rem;
```

Comment:
if we can define all vars as `!default` override may easy.
```scss
$oxd-switch-width: 2rem !default;
```
or you can find best way

Reply:
no let's do it this way, i've added it now